_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,366 @@
+
+This strategy complements existing IT initiatives, such as [ESDC IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), and supports them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) regarding usage of the Version Control Systems.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Products are to be built.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- ESDC Information Management Policy (being drafted)
+- ESDC Security Policy (being drafted)
+
+### Development and Operations Teams (incl. SSC) <!-- omit in toc -->
+
+1. Use a DevOps platform, that includes a Git-based Version Control System (VCS), among the following options, in that order of preference:
+   - Public (Unclassified): **[GitHub](https://github.com)** or another [approved public platform (internal)](https://dialogue/grp/TAWS-ATST/Technology_Standards/Brick%20Repository/Software%20Version%20and%20Source%20Code%20Management.mht)
+   - Interdepartmental (up to Protected A): **[GCcode (internal)](https://gccode.ssc-spc.gc.ca)**
+   - Internal (up to Protected B): **[Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/)**
+2. Follow the guidance of [ESDC Open Source Software (OSS) Management Framework (internal)](https://dialogue/grp/TAWS-ATST/Collaboration%20Architecture%20Services/Projects/Artificial%20Intelligence%20and%20Open%20Source/OSS/ESDC%20Open%20Source%20Management%20Framework%20v%201.1.docx).
+3. Leverage the functionalities provided by the selected VCS as much as possible, for instance:
+    - File hosting (source code, scripts, system/database configurations...);
+    - Issues (incl. milestones and Kanban);
+    - Wiki (documentation);
+    - Build and Test Automation (e.g., security-related); and
+    - Deployment to internal or external platforms.
+4. Make their projects in the VCS open by default and adhere to best practices such as:
```

Comment:
I would split it into two:
4. Make their projects in the VCS open by default (do we mean read-only? Or also contribution as well?)
5. Adhere to best practices such as:
- ...
- ...
- ...

Note that last 5. best practice could also be an action item in the coherent set of actions (Architecture/Define best practices and standards). I.e.
5. Adhere to best practices as listed in the <ESDC best practices and standards> (which is something to write)

Reply:
Working in the open means the team maintains the control on what makes it into the project but supports and encourages contributions... I would propose that this is the default behavior :thinking:

Reply:
Great suggestion! Added to the document. I agree that contributions must be encouraged by default.